src/test/vmmalloc_valgrind/TEST1

```diff
@@ -51,10 +51,6 @@ set_valgrind_exe_name
 configure_valgrind memcheck force-enable $TEST_LD_LIBRARY_PATH/libvmmalloc.so
 setup
 
-if [ $(valgrind_version) -ge 312 ]; then
-	export VALGRIND_OPTS="$VALGRIND_OPTS --soname-synonyms=somalloc=nouserintercepts"
-fi
```

Comment:
Why do you remove it?

Reply:
This is already done in unittest.sh:708 

`if [ "$preload" == "libvmmalloc.so" ]; then
			export VALGRIND_OPTS="$VALGRIND_OPTS --soname synonyms=somalloc=nouserintercepts"
		fi`

Reply:
ok